test/test-server/kafka_rest_router.go

```diff
@@ -235,6 +235,8 @@ func handleKafkaRPTopics(t *testing.T) http.HandlerFunc {
 	}
 }
 
+func PtrString(v string) *string { return &v }
```

Comment:
cpkafkarestv3 v0.3.15 removes PtrString. Thus introduce the function.

Reply:
Any idea why it was removed? It would be nice not to have to write it.

Reply:
No idea. The function is defined in `utils.go` in `kafka-rest-sdk-go`. Looks like `kafka-rest-sdk-go` uses
https://github.com/confluentinc/openapi-generator to auto generate the file. Somehow other commits in `kafka-rest-sdk-go` do not generate `utils.go` except @linouk23 [PR](https://github.com/confluentinc/kafka-rest-sdk-go/commit/ad2863a3d82e67b14c98be24ef2b45901de2ac05#diff-3641002196497f9f2eca0070a878d4ee147ef4a7938640be667780ab7300d12a).
@linouk23 Did you do something special to get the auto-generated `utils.go` in `kafka-rest-sdk-go`?

Reply:
Created https://confluentinc.atlassian.net/browse/API-1204 for this. Since @linouk23 is working on other stuffs now, going to merge this. Once he fixes the issue, we can revert this change.